README.md

```diff
@@ -95,8 +99,10 @@ Review the following use cases to understand how to use Zowe Explorer.
 You can use the following functionalities when interacting with data set:
 
 * **View data sets and use multiple filters**: You can view multiple data sets simultaneously and apply filters to show specified data sets.
+* **Rename data sets**: You can rename specified data sets.
+* **Copy data sets**: You can copy a specified member inside a dat set.
```

Comment:
Copy works for data sets and data set members

Reply:
Typo 'dat set'

Reply:
@jalel01 nice catch! thanks.
@crawr Do you mean that it's possible to copy an entire data set?

Reply:
@IgorCATech yep. you can copy datasets. I tried copying Sequential to Sequential and Sequential to PDS Member v.v.

Reply:
oh ok... so not all types of data sets are possible to be copied. Thanks for the answer.